### ISSUE TITLE
Account / User profile: Limit/Forbid specific user names

### ISSUE THREAD

**Issue by andreoda** [VocaDB/vocadb#1713]
To prevent mix ups and for proper moderation.

User names:

- Admin
- Administrator
- Support
- Manager
- Owner
- Founder
- System
- Moderator
- Mod
- Developer
- Dev
- Staff
- VocaDB
- Vocaloid Database
etc.

Perhaps, this list could also include some other names like those related to profanity.
Feedback is wanted.

Related: https://github.com/VocaDB/vocadb/issues/1712 & https://github.com/VocaDB/vocadb/issues/1391

**Comment by FinnRG** [comment]
@andreoda This isn't difficult to implement, but I would like to point out that filters like these can usually be bypassed very easily (especially if the list is hardcoded, and VocaDB is open source) and can block normal usernames (i. e. blocking "Dev" could also block one of the many normal names starting with "Dev"). 

I personally think that manual moderation would be more effective. Why don't we create a guideline that misleading names are forbidden and rename the users who create these accounts?

This also applies to filtering profanities (I very much recommend this video by Tom Scott about content blocking: https://youtu.be/CcZdwX4noCE).

But I'm open to other suggestions/feedback regarding this.

**Comment by andreoda** [comment]
That sounds like a good idea. Maybe we could also use a bot to check for forbidden usernames when they register or when they rename themselves to such.

**Comment by CatgirlFrostmoon** [comment]
Yeah, direct word blocking is often sketchy - many of these are common enough in normal usernames, especially if someone's a dev for a separate site. Blocking profanity / slurs are definitely less likely to run into major issues, as long as you keep it to just definite cusses (i.e. fuck, shit) or definite slurs. Adding acronyms or double-meaning words (i.e. pussy, CP) would be an incredibly bad idea, conversely.
